test/fixtures/friendships.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  id: 1
+  user_id: 1
+  friend_id: 1
+
+two:
+  id: 1
+  user_id: 1
+  friend_id: 1
```

Comment:
Why did you add information to the fixture if you don't actually have a test that uses it?